### DISCUSSION TITLE
EU Funds (BG-61)

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#439] [Q&A] [answered]
Good Morning Lab !
In the doc I found the paragraph about EU Funds (BG-61), but in notice subtype 7 ( pin time limit ) I only find a combo where I can choose if the procurement is at least partially financed by European Union funds (BT-60-Lot) .
I did not find , for example BT-7220-Lot in 7.json. 
UI attached
![image](https://user-images.githubusercontent.com/52211037/236128818-df6546c6-14ec-433e-b110-08e2d6142d9e.png)
Bye
Renato

**Comment by rouschr** [comment, marked as answer]
Internally we have a ticket to add these "new regulation" fields in the notice-types (TEDEFO-1865), we are currently lagging behind as each new field must be added at the correct location in one or more notice types (up to 45).
These will probably be part of SDK 1.9
In the mean time, if you know the extact location, you could modify the 7.json and add a field there by yourself, the issue would be that if you later move to SDK 1.7 or 1.8, you would have to keep/redo your changes.

**Reply by TateGunning** [reply]
Will you confirm .7 location please

**Reply by rouschr** [reply]
Do you mean 7.json or SDK 1.7?
https://github.com/OP-TED/eForms-SDK/blob/develop/notice-types/7.json (for a specific release see Tags next to the branch selection.
SDK 1.7 is not released yet.

**Reply by TateGunning** [reply]
Just pushed some buttons on 1.6, but yes you read my typo correctly I was referring to 1.7. What just happened on 1.6, was it good or bad?

**Reply by rouschr** [reply]
@TateGunning  SDK 1.7 is not officially out yet, this is why I said "later".
You can have a preview at 1.7 (unstable) here: https://github.com/OP-TED/eForms-SDK/tree/update_sdk_1.7.0
